### PR TITLE
Unified Properties for Student & Grading Edition

### LINKED ISSUE #196: `Feature`: Allow use of PAT (for 2FA) / Introduce Advanced Settings
**Is your feature request related to a problem? Please describe.**
Followup to #194 . In the student edition we should add a checkbox "Advanced Settings" or something similar to keep the config as simple as possible for students.

**Describe the solution you'd like**
Advanced Features Property View (or Checkbox) and possibility to enter a PAT (Personal Access Token) for the GIT Interactions.

### PR DESCRIPTION
**Pull request by dfuchss**
Resolves #196 and removes not used relative config path

### CODE REVIEW

File: bundles/edu.kit.kastel.sdq.eclipse.common.core/src/edu/kit/kastel/sdq/eclipse/common/core/GradingSystemwideController.java [hunk 1]
```diff
@@ -317,16 +317,7 @@ public void submitAssessment() {
...
-			if (this.course != null && this.exercise != null && this.submission != null) {
-				// not the case at startup with rel config path chosen!
-				this.setConfigFile(new File(ResourcesPlugin.getWorkspace().getRoot().getProject(this.getCurrentProjectName()).getLocation().toFile(),
-						this.preferenceStore.getString(PreferenceConstants.RELATIVE_CONFIG_PATH)));
-			}
-		} else {
-			this.setConfigFile(new File(this.preferenceStore.getString(PreferenceConstants.ABSOLUTE_CONFIG_PATH)));
-
-		}
+		this.setConfigFile(new File(this.preferenceStore.getString(PreferenceConstants.ABSOLUTE_CONFIG_PATH)));
```
Comment (Shirkanesi) [inline bundles/edu.kit.kastel.sdq.eclipse.common.core/src/edu/kit/kastel/sdq/eclipse/common/core/GradingSystemwideController.java:320]: Never used the relative config; just curious why was it removed?

**Comment by Shirkanesi** [comment]
@dfuchss is this PR currently working on your system? For me (win11-Edu) is it not working:
grafik

**Comment by Shirkanesi** [comment]
LGTM 👍

File: bundles/edu.kit.kastel.sdq.eclipse.common.core/src/edu/kit/kastel/sdq/eclipse/common/core/GradingSystemwideController.java (hunk 1 above)
Comment (dfuchss) [reply bundles/edu.kit.kastel.sdq.eclipse.common.core/src/edu/kit/kastel/sdq/eclipse/common/core/GradingSystemwideController.java:320]: We do not use the feature and as we have no use case for it currently it has been removed . It was an old idea .. maybe we'll add it somewhere in the future again

**Comment by dfuchss** [comment]
I have to look at it. But afaik no big changes are made. I'll look into the deps in more detail.

**Pull request merged**